resources/server/bin/code-server-linux.sh

```diff
@@ -21,7 +21,7 @@ if [ -f "/tmp/vscode-skip-server-requirements-check" ]; then
 fi
 
 # Check platform requirements
-if [ "$(echo "$@" | grep -c -- "--skip-requirements-check")" -eq 0 ] && [ $skip_check -eq 0 ]; then
+if [ "$(echo "$@" | grep -c -- "--skip-requirements-check")" -eq 0 ] && [ $skip_check -eq 0 ] && [ -f $ROOT/bin/helpers/check-requirements.sh ]; then
```

Comment:
The cleanest is to no have that check at all. 
I suggest to create a `code-server-linux-legacy.sh` file and let the build script pick one or the other.

![image](https://github.com/microsoft/vscode/assets/6461412/052fd6ba-1c9e-44ce-9172-9481c10e62d2)

You can also add some additional echo there that warns that this is a legacy build that is unsupported.